client/src/components/Button/Button.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+const Button = (props) => {
```

Comment:
A nice to have would be to write your components like so:

```
function someComponent(props) {

  return (
    <div></div>
  );
}
```

Just the recommended way the React devs suggest

Reply:
I have refactored and will push the changes tonight.